src/routes/posts/posts.controller.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const { promisify } = require('util');
+const writeFile = promisify(fs.writeFile);
+
+const postsData = require('../../db/data.json');
+
+const PATH_TO_DATA = 'src/db/data.json';
```

Comment:
Is this not referring to the same file as the above const `postsData`?

Reply:
It refers to the same file, but the path used in writeFile is an absolute path - tried to use a relative path for the writeFile and it didn't work :(

Reply:
Oh, interesting. I see that they've done the same thing in the boilerplate: https://github.com/bridge-school/be-cohort1-sample/blob/2.3/error-handling/src/routes/movies/movies.controller.js#L25 

I wonder if this would help? https://nodejs.org/api/fs.html#fs_fs_realpath_path_options_callback

Reply:
Sounds great! Going to look into it :)